### PR TITLE
Client management

### PR DESCRIPTION
**Pull request by jacinyan**
## Rationale

This is for merging updates on Epic 4 Client Management

## Advice for Reviewers & Testing Notes

- In 'src', a folder named 'redux' has been created, followed by 'changeState.js' being moved under one of its sub-folders, yet not default functionalities have been tampered with
- In 'views', a folder named 'client_management' has been created
- Various items at the sidebar were hidden
- Library name at the top of sidebar was removed
- Website document title has been changed
- Put back @coreui/icons

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

## Task Name

- Issue 46

## Linting Checklist
- [ ] No commented code
- [x] Code Formatted nicely (Prettier)
- [x] PR your own code before you assign reviewers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,12 +36,12 @@
   "dependencies": {
     "@coreui/chartjs": "^2.0.0",
     "@coreui/coreui": "^3.4.0",
-    "@coreui/icons": "^2.0.0",
```
Comment (lucitemple) [inline package.json:39]: @jacinyan is there a reason to delete the icons? Are we using a different icon set? I have used some of these.

Comment (jacinyan) [reply package.json:39]: Oh, I think I was trying to add @coreui/icons-react and perhaps have them accidentally deleted while I was typing in the command.Feel free to add the @coreui/icons back. Cheers~

**Review by lucitemple — COMMENTED** [review]
@jacinyan I've reviewed what I can, but I don't know about Redux so can't properly "review" that content.

I had one question about the deletion of the coreui/icons in the dependencies - are you sure it's not used anywhere in the project?

Do you want to get someone who knows redux to review the redux files, or for me to pull the changes as is & hope there are no mistaks?

File: package.json (hunk 1 above)
Comment (lucitemple) [reply package.json:39]: if you can quickly add it back in, and push your branch, and let me know, I will pull it.

Comment (jacinyan) [reply package.json:39]: @lucitemple  And I can't seem to respond to the comment you left below. Gift seems to have reviewed the PR yesterday and waiting for your confirmation of using Redux. Cheers

**Review by lucitemple — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lucitemple) [reply package.json:39]: ok, so maybe next time he can review it on github and put the review through here? merge done now :)

Comment (lucitemple) [reply package.json:39]: ok, so maybe next time he can review it on github and put the review through here? merge done now :)

**Comment by jacinyan** [comment]
@lucitemple A message pops up saying it's safe to delete the branch now. Can I do it now?

**Pull request merged**